backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -417,6 +423,27 @@ private ApiUser getCurrentApiUser() {
     }
   }
 
+  private ApiUser getCurrentApiUser() {
+    if (RequestContextHolder.getRequestAttributes() == null) {
+      // short-circuit in the event this is called from outside a request
+      return getCurrentApiUserNoCache();
+    }
+
+    if (_apiUserContextHolder.hasBeenPopulated()) {
+      LOG.debug("Retrieving user from request context");
```

Comment:
should this show more identifiable info? is it even needed?

Reply:
It was necessary for development! It's consistent with the analogous debug log in `getCurrentApiUserFromIdentity`, and won't appear in prod – where I believe the default log level is info

The logger included with _creating_ a user includes identifying info, but otherwise they're not identified 

So.. I think this is fine as-is